### DISCUSSION TITLE
Driver TFT ST7735 SPI

### DISCUSSION THREAD

**Discussion by sheritafita** [arendst/Tasmota#16180] [Q&A] [answered]
Hi Everyone,

I would like to interface an LCD ST7735, SPI with an ESP 32 but Tasmota seems not to handle it yet.
If ever someone made it works, I'm interesseted please.

Thanks in advance

**Comment by sfromis** [comment, marked as answer]
I'm not using that display, but since there is a [descriptor file](https://tasmota.github.io/docs/Displays/#descriptor-file) for it, it should be supported by Tasmota, within the uDisplay framework at least.
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/ST7735S_M5StickC_display.ini

**Reply by sheritafita** [reply]
Thanks @sfromis 
I gave it a try but didn't work with M5StickC_display.ini. 
I used ILI9341_display.ini instead and had to adjust x,y offset

I have just 2 details to fix, if ever someone can help : 
- It's flickering when I'm on displaymode 1 (time display)
- the brigthness is too low

**Reply by GVollaro** [reply]
If anyone is interested, I attach the ".ini" file, created starting from the ILI9341 and ST7789, for the ST7735 display (different from the ST7735S) that I just tested.
[ST7735_128x160_display.zip](https://github.com/arendst/Tasmota/files/11592240/ST7735_128x160_display.zip)

**Reply by s-hadinger** [reply]
I just added it in the repo (as discussed on Discord), thanks @GVollaro

**Reply by jkwim** [reply]
> If anyone is interested, I attach the ".ini" file, created starting from the ILI9341 and ST7789, for the ST7735 display (different from the ST7735S) that I just tested.
> [ST7735_128x160_display.zip](https://github.com/arendst/Tasmota/files/11592240/ST7735_128x160_display.zip)

I bought a what was supposed to be `ST7735` 1.8" 128x160 display from Aliexpress.
https://vi.aliexpress.com/item/1005008346811816.html

Your configuration:
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/`ST7735_128x160_display.ini`

worked for my device.
